scripts/token-paymaster-v2/deploy-token-paymaster.ts

```diff
@@ -0,0 +1,128 @@
+import { ethers, run } from "hardhat";
+import {
+  deployContract,
+  DEPLOYMENT_SALTS,
+  encodeParam,
+  isContract,
+} from "../utils";
+import { Deployer, Deployer__factory } from "../../typechain-types";
+
+const provider = ethers.provider;
+const entryPointAddress =
+  process.env.ENTRY_POINT_ADDRESS ||
+  "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
+const verifyingSigner = process.env.PAYMASTER_SIGNER_ADDRESS_PROD || "";
+const DEPLOYER_CONTRACT_ADDRESS =
+  process.env.DEPLOYER_CONTRACT_ADDRESS_PROD || "";
+
+function delay(ms: number) {
+  return new Promise<void>((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, ms);
+  });
+}
```

Comment:
This should be managed in a central `config.ts` and not duplicated across scripts

Reply:
DEPLOYER_CONTRACT_ADDRESS and entryPointAddress are applicable to more scripts. Have left them as it is to have control on individual script. 
moved delay to utils/